### PR TITLE
Bug: Register OpenTSDB with correct service name

### PR DESCRIPTION
**Pull request by idafurjes**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM

File: pkg/tsdb/opentsdb/opentsdb.go [hunk 1]
```diff
@@ -53,7 +57,7 @@ func (s *Service) Init() error {
 		QueryDataHandler: s,
 	})
 
-	if err := s.BackendPluginManager.Register("opentsdb", factory); err != nil {
+	if err := s.BackendPluginManager.RegisterAndStart(context.Background(), "opentsdb", factory); err != nil {
```
Comment (wbrowne) [inline pkg/tsdb/opentsdb/opentsdb.go:60]: It's sadly unclear from the API but this should be addressed soon - but Register is most appropriate here 😀

Comment (idafurjes) [reply pkg/tsdb/opentsdb/opentsdb.go:60]: ok, at this point influxdb, graphite, loki, tempo and cloudwatch has RegisterAndStart, is that a problem?

Comment (wbrowne) [reply pkg/tsdb/opentsdb/opentsdb.go]: No it's okay - under the hood they do the same thing. We can have a separate PR for that

**Pull request merged**